src/v1.js

```diff
@@ -349,6 +347,10 @@ api.declare({
   stability:    'experimental',
   description: [
     'This endpoint will trigger the creation of a task from a hook definition.',
+    '',
+    'The HTTP payload must match the hook\s `triggerSchema`.  If it does, it is',
+    'provided as the `payload` property of the JSON-e context used to render the',
```

Comment:
payload? Ok, make sense if you are thinking in get some info, but is it a JSON-e context, right?

Reply:
It will be a sub-object of the JSON-e context:
```
{
  firedBy: "triggerHook",
  payload: payload,
}
```

Reply:
Sure.